gitops_workloads_infranodes/running-gitops-control-plane-workloads-on-infrastructure-nodes.adoc

```diff
@@ -11,15 +11,8 @@ You can use infrastructure nodes to isolate infrastructure workloads for two pri
 * To prevent billing costs associated with the number of subscriptions
 * To separate maintenance and management
 
-You can use the {OCP} to run {gitops-shortname} control plane workloads on infrastructure nodes. This includes the Operator pod and the control plane workloads created by the {gitops-title} Operator in the `openshift-gitops` namespace by default, including the default Argo CD instance in this namespace. 
-
 With {gitops-shortname} control plane workloads, you can securely and declaratively isolate the infrastructure workloads by creating multiple isolated Argo CD instances in a cluster, with full control over what an Argo CD instance is capable of. In addition, you can manage these Argo CD instances declaratively across multiple developer namespaces. By using taints, you can ensure that only infrastructure components run on these nodes.
 
-[NOTE]
-====
```

Comment:
I've deleted the comment as per your request halawren@redhat.com. Please take a look.